library/std/src/lib.rs

```diff
@@ -249,6 +249,7 @@
 #![feature(clamp)]
 #![feature(concat_idents)]
 #![feature(const_cstr_unchecked)]
+#![feature(const_transmute)]
```

Comment:
This needs bootstrapping annotations; on higher stages you need `const_fn_transmute`.

Reply:
I find that `#[allow_internal_unstable(const_fn_transmute)]` allow the code to be built.